storage/src/main/java/com/tencent/rss/storage/common/HdfsStorage.java

```diff
@@ -106,7 +106,7 @@ ShuffleWriteHandler newWriteHandler(CreateShuffleWriteHandlerRequest request) {
           request.getEndPartition(),
           storagePath,
           request.getFileNamePrefix(),
-          request.getConf()
+          conf
```

Comment:
why use this conf?

Reply:
This is the actual conf pass from HdfsStorageManager, which may be applied the sepcific  remote  storage conf.

Reply:
This conf is useless?